compiler/onecc-docker/tests/onecc-docker_002.test

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+# Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# run a workflow where one-import-tf generates intermediate files
+
+filename_ext="$(basename -- $0)"
+filename="${filename_ext%.*}"
+
+trap_err_onexit()
+{
+  echo "${filename_ext} FAILED"
+  exit 255
+}
+
+trap trap_err_onexit ERR
+
+workflowfile="onecc-docker_002.workflow.json"
```

Comment:
`workflow` is WIP. ~let's remove this test.~ let's not add this now.

from @seanshpark https://github.com/Samsung/ONE/pull/9793#discussion_r979707160

Reply:
The `workflow` is not ready to use yet. So we don't need to implement it.

from @jyoungyun https://github.com/Samsung/ONE/pull/9793#discussion_r979712649